src/constants/categories.js

```diff
@@ -826,7 +826,7 @@ export const CATEGORIES = [
         charts: [
           {
             name: 'Workers in knowledge intensive services',
-            axisTitle: 'Percentage of employment as knowledge workers',
+            axisTitle: 'Percentage of employment',
```

Comment:
This axis title is "Percentage of employment" but the similar chart on the jobs page "Industry sector share of employment" has the axis title of "Proportion of employed persons". Should they be the same as each other?

I'd say it's a percentage, not a proportion, and I don't know if we're standardising on 'people' or 'persons'.